packages/client/src/exchanges/getExchangeBookRequest.ts

```diff
@@ -0,0 +1,30 @@
+import { adaptError } from '../helpers/client/formatError.js';
+import client from '../helpers/client/index.js';
+import join from 'proper-url-join';
+import type { GetExchangeBookRequest } from './types/index.js';
+
+/**
+ * Method responsible for getting a specific exchange book request.
+ *
+ * @param id - Exchange identifier.
+ * @param bookRequestId - Book request identifier.
+ * @param config - Custom configurations to send to the client instance (axios).
+ *
+ * @returns Promise that will resolve when the call to the endpoint finishes.
+ */
+const getExchangeBookRequest: GetExchangeBookRequest = (
+  id,
```

Comment:
```suggestion
  exchangeId,
```

I think that is more explicit as to which id it refers to. Change the @param name as well to `exchangeId` please.

Reply:
done 👍